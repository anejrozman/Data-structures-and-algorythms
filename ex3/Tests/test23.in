649,647,127
6.242519466476784,2.1430764725329037,5.6539442422844095,3.2637129857415283,3.721961537539431,7.754133556510062,7.375934375812214,9.894184847843354,4.742549805646214,2.2397593457487117,8.977595253698526,7.878079974410429,3.845857779900448,7.1085926567735145,7.932476951843097,5.0258749216218135,1.1412282458157705,9.68263932368104,6.954942851307641,4.442175605443044,7.133949094681181,6.9346677449408585,6.095802734694986,7.8448796936797045,8.368044239552178,9.07850830582905,7.004754238795367,9.309120503890913,1.9000805387848239,7.808784854929727,6.170031183154538,3.2493148440786244,6.692306083856431,5.784273232760827,5.247512165617191,7.185819472892898,0.7855116541451186,1.5674366757172478,9.497546405560911,7.074474245544101,2.26070652558432,3.5195104083665387,5.976508779493836,6.516818314987799,5.359620226657815,9.372056728870987,2.505000888461244,7.408716289200374,4.057009729248876,3.2194812254918337,2.932627148179847,2.878390424789645,9.100399189371066,5.185449019223411,5.164353036325581,1.9561657717299483,4.9096122485247395,0.936347703253282,7.522314622809138,9.216503759974202,4.753562977327649,8.118228662259671,3.8826551954225685,6.1773509304321,3.8730741350146807,7.259050185691127,5.980561265139479,7.5506970347437585,6.550413341549884,4.9286677701481905,5.350009635196151,9.336186196580375,3.9129667338993124,4.533725512968941,6.572913672662106,4.044292324194797,5.797130754387007,8.70118525393687,6.379203617035855,3.1046056415211463,2.5359780655004833,5.9141478741380205,9.381211712898072,6.602531713073547,3.273819877790982,2.9485031421678545,0.6871920807974667,7.2751524936386165,5.3271340377645355,9.789046798517358,8.407204346129465,6.580081482833462,6.830753063293936,1.6369407375709546,5.743938810167623,0.987333709265998,1.5943761517765749,5.9509180624939475,8.565126147699681,5.763732464506142,9.94125148828877,1.8518103731631796,9.71649671852299,8.418990693944789,0.9416425822106667,5.053183578835644,5.075146052213127,8.335979192062727,2.6344782446523043,2.919479721795327,5.0107301775513085,3.1659118934748087,6.505310741297216,3.777108249209194,2.6825240560491075,5.988151534371463,6.7195268595045645,6.156149547818764,0.5571772406318923,6.312251747198741,5.150558418891182,1.4942875482323466,3.734545269136362,2.6143671723271367,5.094658836263461,4.4664095962765735,5.43246279718634,2.81613955941011,7.55213825933419,1.06727146951974,4.787606153116273,1.3710128307263139,4.551000387410169,1.9853571998845054,2.4375314029347566,1.5430884206693671,8.453634675265661,6.394513257732772,5.975558160285022,9.097889082522856,1.1090434096079904,6.888662562967639,9.477788206130906,8.924846271164519,7.71162818970297,2.5215491412705293,2.0507659108858824,7.008252706303483,5.976599333445372,1.7250720883471142,8.875925001034275,7.290169377914781,9.445137809689031,2.4272605849580584,5.236314179336113,9.50067975768983,3.4391549376933965,3.4803571516479543,9.380016867836039,9.079854400233243,6.654916810951082,4.2106767090602375,5.291445788024083,2.4557186423410777,6.455327582540877,4.143009559944273,4.911665652873246,1.697410552749507,4.9925476100759445,7.44976860708977,6.850416422893778,3.3673379021885474,5.199896217384477,0.7417450227981424,8.263420805094388,1.2916010140846685,6.072423544829779,9.644603084948407,5.16838448032972,3.8357345923876114,2.7997295770186486,5.057791601253553,4.317572907658123,2.6245363647165334,9.854506329536331,8.840884751618146,6.372305495218472,6.268081115768009,7.93538935866674,4.060701381304697,3.857011568594225,3.8154403584486714,7.768297537350077,6.413420646219166,5.4432603478721875,8.121704374360451,2.1896229023914167,5.831676427554431,8.792122629046956,5.942394982313207,6.5310758865375105,6.287342377664131,9.760075949066678,4.316234862513962,6.160511327985999,3.6747250772790725,2.161600312505634,5.791935649096634,8.022523447899106,0.6041568996072473,6.46815170993688,6.289110722651744,0.9283536187646478,9.296979828239438,8.426606085732205,7.606737048418163,1.536202387722034,2.6079441343333474,9.245818710968743,4.830137899550227,2.004278041340692,1.3387367430481771,2.2186719698618917,4.147172434794562,8.373308910245601,3.542055068339898,4.133357678197045,4.9141450552090475,2.71301372850661,1.6235547858201431,4.857389863664563,9.832443665505759,9.63181949844377,7.464648060001743,2.329789658736416,9.398774703077974,1.1180635223766118,0.8760385583154596,1.6656108759340658,9.166096584766436,8.009553581727175,2.7269936995941704,4.371994230412198,1.862095759860385,1.7184557738636719,7.7333405429760385,1.6750856335420599,9.455377528742497,2.3168214488409857,9.663081045798725,6.777833838963262,2.0986006680591163,4.613651103375628,8.137306113833255,3.6577509160491464,9.821384371777958,7.9569519625884455,3.5072097288476125,5.097323206617897,1.0693475569403086,6.428916492362445,4.966009244736158,4.26422513340544,9.092435263823521,9.682523359192983,2.33191581241263,5.141877419847697,1.8831490867291323,3.5463146002105566,2.533958284425891,4.783778925881102,7.191334960748014,6.506959435866847,2.627618355881372,4.494133255682126,6.688574354342015,9.50778868723719,7.4842393777040375,3.6791585397812736,6.5916464367285394,6.61346480905275,3.195450811985758,6.828357568213561,8.04137494433836,7.447584352583358,0.6186562898143768,4.811897878754894,6.737884529153803,5.542157605496519,7.662657793723317,6.84118476311252,2.7191414452227836,7.025066913557245,9.186147726481028,0.8506330719177113,6.733108384336902,6.127927402973889,1.9826457016587296,2.6331303991644557,4.833247482185985,5.903514823915294,1.7417817160140274,7.1120182442364746,9.040090507478437,9.853168992508056,0.6613999637975584,4.873016749299099,8.752950027040267,6.740376393115725,1.9320947085056848,6.871133819289439,5.746584211594024,8.457711467716416,5.688639683507734,0.9734119207244354,9.099081510357584,9.968724499857503,9.181798563709563,5.789485547092993,7.362593885558545,9.199084385225303,9.490082734317264,9.101946704677562,7.682056201623804,8.041908810575734,3.104514145407526,8.336531583400001,5.75354055443681,1.3734923158724548,9.094170016117856,8.626058712484635,6.895958970106373,8.422428142338248,8.089121235037634,0.9908893847230826,5.689287886653888,1.1449969189163456,9.296066832043236,3.281124660952349,3.34744826122468,8.43486947367917,2.6598971649896086,4.159588126454576,7.136981170100948,9.384176922960332,9.344221655102881,1.3962271553083534,7.59120464271719,3.417965453422945,7.998430550813703,8.884547416324615,9.34267900787039,4.029954176393124,4.545617778955128,3.985819697298289,9.451929131241426,9.503040507626107,4.55962697498744,4.075493187411136,0.5514195023362578,9.339794924573292,2.02157380526502,5.927120015711494,3.526508469303969,3.3726360197497254,2.9103322703108216,5.734134021975846,9.552541544848442,7.472692026160968,3.960498968541392,0.6301141646232764,6.984008315281067,5.818827312841581,5.083042830138256,1.1116224078696586,5.95617085529068,6.2052193918893375,9.547653260059601,6.149667164117159,7.466264707716942,9.32334852561171,8.01220129144616,7.570384445953281,5.807142960561993,4.341206549406283,3.9449567337456983,4.511213862803004,1.3706127366692367,1.2620977866929812,5.864663938887307,1.9518431840749424,5.700707826740792,5.9325461960430745,8.993389594984517,9.890710213187436,3.28157942584993,8.233065910502562,1.9665341807813568,8.357305446752541,7.188436883105975,2.8427173734981803,2.333927168189274,4.965157204840963,6.888955083254526,2.70011459320093,9.792856461168899,9.270873651496498,0.7222586867591165,3.1811526502282046,2.1124175408303296,5.803859634736398,8.716139813088272,5.375725781193684,6.570449385052363,5.795688748687575,3.2336187241339305,2.038793808574159,2.707357505826977,2.39456441401664,6.445479925133499,2.469971728139207,8.806602937768616,5.752598594215733,0.9212078295103229,7.0587287102440985,0.6129863270969854,7.566258629205625,7.36537163002889,4.96618390667917,9.344073307840878,8.888870690906975,9.516327173640093,2.3160121525562323,7.724421501140731,3.7923334910983635,4.725294995305651,8.11965291722815,3.2573030705749297,0.7504577567448822,9.313153506328227,1.5920503499122958,7.088780844979121,7.780834417497338,7.25660659543867,3.283851883201852,2.7295317620647896,2.724244458874267,1.1104724744184924,9.06731344897019,5.5765391740549175,5.094214531287538,3.7237009209711784,8.70080337641381,9.244953182533227,4.520480309803002,1.150994761441396,1.8573319329663447,1.4444971467477048,9.932291657662411,5.827540025106415,1.2813861005896914,1.8054522618103803,6.079894236007598,3.516014950424994,5.22499465286787,7.285684599959635,8.888143647419136,2.5027353733209545,7.6208695507640245,5.048081898828392,2.2408509930249787,4.863084022556138,8.538640088476747,2.4151748776447333,1.9924983309801791,9.143325990808448,6.81128943688892,3.8164237549259177,1.623123714123102,5.597626594291967,8.115203463915762,4.477604473288839,5.919370850505177,3.211798737841752,9.941010612507062,9.844179515946028,3.7034256124784477,6.924416098500468,3.1455338754980784,5.158882111632034,1.0849853966734666,1.6485317430232038,2.0503537341782,1.5125036786979562,8.788910767535564,3.254717746726733,9.46303581231561,2.8498808000200353,9.079141506207264,8.751873501543608,5.528628073577045,6.820497333486443,4.70081979708397,4.319340891388364,9.587429923795225,5.3711309614044485,0.9318464492089554,7.484190945960615,8.984423397534416,2.6216194068412904,8.700679147321642,6.282897818247421,2.732710393140635,1.988425109882518,3.009206524660119,4.0620922491776295,5.36500079459629,0.9409161884626365,1.6463586180614762,8.89927364504129,3.5467126448186903,1.1378767985958866,9.70928283219311,8.306093628589148,4.343874920359204,7.716055734629206,4.19917348926689,3.1177137682899936,7.603535779914375,2.9831864586523085,3.2675041019368676,0.5868231543020919,6.489318530543718,9.302523219138799,2.6473063344407177,9.049035287614174,8.163351330732205,1.274956134343228,2.4591643862815173,8.402765985856679,7.293774149325759,2.5834989013529164,7.136569257836711,7.063948687381254,6.296744128299034,5.230502525056616,2.5071344089798444,6.990526368616893,7.050463745225401,8.464047230625047,7.405139872130882,1.4433050303431945,8.776908496610707,3.477792173681554,2.8181682996279616,3.4002174678412134,1.953008649919945,3.990756343749248,3.0709896011036766,3.183215672007732,6.073399648436749,3.604473848330668,1.34426961762557,2.9851176857767197,2.890417876065917,7.141311787733968,6.034608296976245,9.681264589456335,5.407026256596896,3.064577820304843,9.249592630008744,2.005318612536204,0.9424157208376428,1.1456269490884121,5.555622617768158,5.035134487261442,9.548248539092762,4.492134860637448,3.704461627764709,5.570953275292414,6.068230295073385,7.802073899522665,5.4694187672679835,9.454652902979916,8.65699491715955,9.647052199893373,4.9497040473972955,4.450108532508683,4.641300549511008,3.0754854670018714,2.8023057578101946,2.6284602804869968,2.462787056178467,9.075497022303425,9.592231024427365,8.652528895649244,1.3188964578392979,7.2139924352476665,3.443408854318815,8.320409219895964,7.1424750289128935,8.531840035873957,7.693227354476945,7.419321501775793,1.3637917877117405,5.241903389889739,3.2513126550438476,7.439461302245343,1.8545839921452507,1.0020849176435735,9.668567104257228,9.780271496974505,7.189276552039525,9.973513444720497,7.259954932487256,7.844166241910592,1.931799936716926,7.8144112777877375,4.718633017557894,2.2705840920647984,9.048503435510579,5.9282824889607815,7.620528130263227,6.605682250926047,5.456700944480102,8.954506282942734,7.155109397692328,1.2134487622692292,2.879760541772149,2.2689513542614295,6.729740927292126,4.703767591072133,3.5957100435367475,2.732663178173441,4.7112839657522,6.599661929925866,1.3020427215007757,3.9934576591267588,8.330476524909761,8.925053277712736,1.6435376688842438,1.596387355652226,8.998413993905855,4.204375362356801,8.64272684232134,4.589827038340217,5.236675286177473,6.9665955245128455,3.214049953259828
16,625,8.494
176,502,9.749
505,101,10.742
307,359,9.676
285,468,15.795
198,591,15.811
229,182,17.14
460,339,17.973
206,203,19.443
528,118,20.375
89,271,20.843
355,546,22.67
331,52,23.203
40,224,19.855
365,154,21.977
377,577,26.873
343,408,27.986
7,641,29.265
343,458,31.346
29,86,35.791
257,38,36.477
196,371,36.473
157,327,39.67
269,43,39.029
593,71,41.613
488,105,39.583
575,483,43.541
278,474,44.948
464,456,46.157
328,457,46.66
630,627,49.286
129,502,50.561
81,335,49.128
32,528,52.493
646,622,57.006
188,413,58.855
590,378,57.784
124,646,57.342
428,544,61.135
647,591,62.013
489,129,66.363
377,93,66.444
512,207,68.818
576,300,69.963
485,331,68.411
207,384,71.47
324,214,67.859
361,250,74.265
215,130,74.545
485,600,77.913
95,413,81.814
103,272,83.241
150,381,83.907
6,586,87.772
347,634,89.453
535,573,91.187
164,424,92.539
4,530,93.708
461,489,94.593
112,23,94.035
460,447,96.829
178,275,97.43
532,158,94.218
93,42,101.54
302,363,101.405
408,97,105.567
161,478,106.106
67,367,107.154
251,625,107.786
522,400,106.497
634,265,110.722
233,29,107.335
624,219,112.996
347,22,112.001
374,518,114.054
42,483,115.348
134,261,116.515
592,535,117.803
403,292,117.928
173,143,122.298
643,100,122.304
578,593,125.027
555,347,126.391
499,360,126.527
211,168,126.147
193,537,129.056
253,646,130.585
105,205,129.724
79,217,131.691
582,275,133.483
468,325,133.684
125,68,136.195
527,54,140.261
57,287,142.485
109,72,144.919
120,522,146.729
534,256,147.634
32,125,147.315
278,405,149.567
241,558,150.111
24,581,152.646
520,354,152.027
624,298,149.158
253,167,156.749
346,479,157.55
348,315,158.796
105,639,156.704
75,228,156.058
236,478,159.469
242,528,167.385
188,361,168.475
332,187,168.659
113,212,167.884
212,81,171.678
412,130,172.174
233,7,178.275
627,325,179.658
449,188,182.792
249,598,182.447
159,637,184.434
178,235,185.927
232,240,185.255
416,306,187.687
268,234,187.488
565,4,183.586
80,464,190.14
263,215,191.826
56,548,191.027
232,470,193.783
193,319,195.202
316,29,196.884
636,612,197.646
57,611,195.045
412,529,200.097
496,236,201.504
205,549,203.831
491,226,205.848
632,437,205.936
216,500,205.724
193,208,212.886
382,233,214.893
49,21,214.34
176,573,217.072
378,64,219.462
130,506,220.429
330,57,219.429
331,209,222.6
601,11,223.931
208,56,224.852
59,418,225.372
424,226,222.993
152,264,227.295
115,226,227.758
298,291,230.715
119,540,231.691
546,413,232.021
154,502,232.92
328,431,233.244
373,297,235.947
221,82,235.205
437,219,237.556
643,130,241.903
23,625,242.973
208,164,242.764
36,632,242.925
78,80,245.056
332,212,245.19
124,382,247.97
300,429,245.65
418,121,251.954
203,80,251.388
546,27,253.325
181,4,253.987
306,232,252.431
47,152,255.496
615,423,253.461
503,537,258.881
32,436,258.933
601,511,258.607
591,300,259.272
248,638,263.046
194,352,265.094
578,317,268.681
489,568,271.963
434,483,272.782
19,588,274.739
359,453,278.235
442,333,278.253
521,551,279.83
629,560,281.281
548,460,280.589
20,4,283.206
114,22,284.663
220,316,288.33
338,100,289.336
138,430,289.467
115,294,288.025
387,608,292.5
491,88,292.086
159,434,293.337
322,326,295.147
241,307,297.579
487,613,298.296
212,266,300.468
640,123,301.549
227,207,306.61
304,84,307.645
337,195,309.569
645,192,311.082
600,646,311.876
587,465,311.892
569,492,311.427
268,413,316.546
218,250,317.398
525,141,316.504
280,78,319.99
337,309,320.799
576,66,321.591
146,344,322.88
219,289,323.039
331,451,320.005
111,188,320.872
109,539,328.834
533,612,329.777
92,524,329.446
365,361,330.374
12,600,332.885
359,523,332.482
561,55,332.277
338,562,333.08
223,163,336.387
606,289,337.766
476,422,344.606
273,420,348.136
385,287,350.578
384,632,351.741
471,619,352.812
474,397,350.748
113,89,354.285
414,86,355.499
41,11,356.391
493,126,356.292
125,409,358.432
51,478,359.835
184,535,358.518
323,93,362.984
201,26,363.227
543,448,362.57
647,23,362.506
616,354,364.751
383,201,367.793
128,383,367.275
200,426,370.186
73,306,371.207
392,333,373.766
562,622,373.205
11,405,377.897
612,269,378.141
420,294,379.936
351,118,377.629
69,287,383.303
642,458,383.645
112,24,383.992
526,542,388.386
50,391,390.435
484,479,391.34
411,475,394.91
641,297,394.737
599,563,395.879
376,40,396.739
379,477,399.296
354,299,401.603
389,457,401.269
396,605,401.62
575,224,401.091
328,237,402.537
570,279,406.127
199,417,409.021
37,448,412.914
457,255,413.298
233,135,414.878
151,244,418.411
355,447,420.378
127,274,422.586
356,21,426.646
133,505,427.2
110,23,430.17
546,231,433.622
83,401,433.663
514,90,436.079
575,40,436.105
48,214,438.595
603,251,439.035
279,319,440.402
69,114,442.663
629,624,445.4
378,618,446.758
33,568,448.584
608,95,451.97
325,120,453.108
347,346,455.396
470,154,457.844
547,45,459.047
514,238,460.657
629,129,461.288
166,249,462.018
401,351,463.147
433,333,464.275
439,505,464.914
467,572,464.4
127,189,467.169
37,165,466.649
615,480,469.545
127,29,468.926
111,519,472.282
522,455,474.998
77,543,476.142
15,110,476.604
621,93,479.014
376,370,482.166
535,197,483.347
486,557,482.85
630,274,483.754
3,147,488.484
351,127,489.322
574,177,489.343
93,563,490.793
208,615,498.927
83,523,498.608
538,482,501.467
198,451,503.144
277,601,511.481
167,159,514.809
458,472,515.442
259,35,516.171
51,347,517.806
241,448,518.101
77,75,518.283
647,174,520.267
176,136,521.67
441,200,522.186
463,120,523.98
375,388,522.774
119,548,525.294
489,521,525.038
305,170,527.283
172,353,527.728
322,217,529.499
151,221,530.016
62,469,526.682
388,152,535.077
404,437,538.246
596,143,539.843
374,544,540.373
316,289,541.724
29,380,542.132
176,274,538.592
404,44,542.078
395,580,538.29
365,159,546.264
525,220,547.882
530,637,547.577
546,454,547.784
140,476,550.712
532,57,549.783
128,575,552.896
214,298,551.938
383,210,554.595
280,466,550.402
8,578,558.064
489,574,559.547
640,89,560.271
437,440,561.327
600,133,562.643
365,209,559.439
68,151,560.935
196,246,562.662
407,594,558.829
79,186,564.11
567,33,562.986
233,25,572.896
123,425,574.85
278,354,575.017
7,262,577.684
321,436,578.905
57,160,579.891
497,380,580.814
606,410,581.474
419,421,580.222
602,436,579.022
598,248,583.45
328,67,588.371
591,11,589.705
451,378,590.752
559,94,590.098
542,199,591.166
198,185,593.348
20,96,594.301
230,23,590.033
186,480,594.0
47,129,597.15
132,376,603.34
624,447,605.094
129,220,606.113
285,336,607.573
116,322,608.657
324,595,609.247
101,628,607.949
336,175,613.531
244,316,620.094
28,507,621.07
547,131,624.972
389,150,626.293
97,354,627.276
521,280,628.193
409,160,629.344
403,128,630.189
229,133,633.746
255,121,634.305
181,489,635.199
485,563,634.326
83,305,638.454
644,264,639.119
373,27,638.662
595,363,641.13
166,614,643.85
506,99,645.278
141,540,646.352
237,543,650.609
235,47,652.029
180,262,652.07
371,572,653.162
306,482,656.251
565,570,657.966
323,190,659.74
348,343,659.947
548,336,661.271
41,412,662.214
293,25,662.014
12,129,664.293
347,55,663.529
647,73,665.01
546,69,667.979
530,173,668.584
49,194,668.936
205,338,670.896
384,597,669.793
439,10,672.592
341,150,673.887
591,394,672.524
487,563,675.727
512,332,676.017
629,484,675.279
444,619,675.216
488,82,682.312
295,364,684.433
603,75,685.76
633,552,685.282
448,316,685.212
95,0,690.082
468,549,691.21
194,241,544.789
118,208,690.379
440,30,694.992
384,472,695.287
353,29,698.596
105,637,699.841
47,342,701.627
119,547,701.372
389,99,701.691
246,103,706.677
551,311,707.559
165,426,708.027
253,43,708.196
145,207,708.96
458,407,711.602
493,572,711.987
295,623,713.476
345,264,714.129
245,494,714.444
18,46,717.754
175,78,719.989
100,489,720.822
75,614,723.412
66,635,724.101
80,420,729.424
86,468,732.641
157,428,741.885
108,496,742.505
546,110,744.96
2,626,745.363
275,132,744.918
236,138,747.371
644,335,749.577
216,271,757.901
503,459,758.424
448,141,758.148
382,357,759.475
472,556,757.495
143,310,760.989
76,374,558.895
210,211,765.216
402,43,766.149
124,342,767.438
46,181,768.654
477,244,766.471
408,118,767.651
58,220,771.321
543,162,772.953
27,435,773.686
539,182,771.352
424,287,776.281
265,542,777.877
485,32,781.343
342,641,782.107
169,586,783.814
617,639,789.935
339,396,790.164
342,93,790.173
98,203,790.21
254,399,793.291
489,417,794.214
238,75,795.701
302,117,797.511
215,322,797.324
547,503,799.913
525,608,804.947
103,356,805.109
54,279,811.509
550,32,817.507
65,624,819.982
632,562,821.795
469,459,827.546
536,238,827.612
435,259,829.45
274,296,829.545
290,352,832.202
405,280,835.39
147,112,838.518
31,285,840.792
109,49,842.188
165,579,843.178
167,122,845.166
31,414,852.554
92,302,852.509
152,206,854.231
593,322,853.567
523,132,856.181
143,578,852.993
409,290,853.883
226,315,856.589
488,418,863.363
60,303,866.967
552,533,866.589
69,626,868.041
245,66,869.721
20,537,870.899
241,206,870.634
312,298,878.885
74,24,879.97
374,117,880.353
546,577,879.98
117,419,878.765
349,478,883.997
234,648,884.908
438,600,885.287
537,303,878.639
76,63,884.478
124,105,888.476
55,460,881.427
495,345,882.187
1,341,887.884
13,392,894.579
334,479,895.88
201,354,896.719
437,334,897.858
159,288,898.681
525,105,903.429
164,323,904.539
600,175,908.1
348,636,910.589
456,527,911.559
580,190,912.421
504,64,913.282
597,478,913.827
332,133,921.742
635,44,923.403
73,479,925.648
397,155,926.597
328,135,927.139
375,242,928.961
400,319,930.34
93,378,931.728
167,535,932.519
575,457,934.25
353,633,935.242
162,205,936.968
422,646,934.67
85,45,938.04
495,522,939.187
196,45,940.797
613,149,941.347
37,297,939.867
637,289,944.968
476,241,948.144
40,403,949.571
67,102,950.98
634,425,948.754
186,231,952.619
600,276,953.831
228,110,954.002
563,169,953.242
149,273,952.245
611,225,956.638
390,409,958.284
383,496,955.232
445,19,960.411
359,584,961.068
454,561,962.623
334,27,960.963
225,384,965.89
622,101,965.354
409,543,966.874
114,428,968.608
542,615,969.279
413,566,968.217
461,302,971.276
583,214,972.885
386,298,965.492
27,211,974.977
389,186,975.447
37,362,976.746
589,530,975.306
263,257,978.771
117,478,981.911
347,17,982.816
309,0,983.241
540,325,990.416
541,314,992.958
422,161,993.587
632,248,994.688
32,13,993.626
21,485,995.683
61,537,995.37
174,366,998.161
372,323,667.942
316,321,668.521
