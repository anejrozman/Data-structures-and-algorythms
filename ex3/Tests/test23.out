494,3487166.9089
5,535.3959
3,128.3318
3,109.1527
3,89.4637
3,34.468
2,81.7407
2,79.6945
2,35.6551
2,33.3099
2,32.4221
2,32.2575
2,26.2935
2,21.0645
2,20.6175
2,19.2448
2,18.2842
2,16.7181
2,14.7795
2,14.2206
2,12.1048
2,10.1843
2,6.2835
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
